wgpu/Cargo.toml

```diff
@@ -32,7 +32,7 @@ path = "tests/root.rs"
 
 [features]
 default = []
-spirv = ["naga/spv-in"]
+spirv = ["naga/spv-in", "naga/wgsl-out"]
```

Comment:
@kvark is this fine, or should we separate this feature between web and native (we need the full SPIR-V->WGSL path on web but not native)?

Reply:
Oh right, hmm. We shouldn't impose the cost to all of the spirv users. Looks like we'll need to move the feature out, like `spirv-web`?

Reply:
@kvark ok sounds good.

@hwoodiwiss would you mind making this change too? I guess we could do something like `spirv-web = ["spirv", "naga/wgsl-out"]` or similar. If this isn't clear, please let me know and I'd be happy to help out

Reply:
Cool, this seems sensible, I'll also update the readme  to include reference to `spirv-web` for wasm targets